src/main/java/nl/thehyve/ocdu/factories/PatientDataFactory.java

```diff
@@ -92,7 +92,7 @@ private void setValue(String[] row, Map<String, Integer> columnsIndex, String co
         header.add("Date of Enrollment");
         header.add("Secondary ID");
         header.add("Study");
-        if (registerSite) header.add("Site");
+        if (registerSite) header.add("Site (optional)");
```

Comment:
yes, this was discussed with Marinel and the representative at OC during the custom demo